### PR TITLE
Fix record syntax to be consistent

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb, @snusnu This is ready for review.

File: gen/src/Gen.hs [hunk 1]
```diff
@@ -70,14 +70,14 @@ generatePackage specification TargetPackage{ .. } = do
...
-mkModelTargets OpenApi.Specification { .. } =
+mkModelTargets OpenApi.Specification{..} =
   liftA2
     (<>)
     (fixtureModules components =<< readFixtures)
     (models components)
   where
     models :: OpenApi.Components -> IO [TargetModule]
-    models OpenApi.Components { .. } =
+    models OpenApi.Componentst{..} =
```
Comment (dkubb) [inline gen/src/Gen.hs]: There is an extra `t` here that blocks compilation.

Comment (mbj) [reply gen/src/Gen.hs]: fixed, good catch.

File: gen/src/Gen/Schema.hs [hunk 2]
```diff
@@ -434,11 +434,11 @@ mkRecordField context parentSchemaObject propertyName fieldSchema = do
         Schema.Reference _name      -> mkPropertyType fieldSchema
       where
         expandSchemaObject = \case
-          Schema.SchemaObject { type' = Just _ } -> mkPropertyType fieldSchema
-          Schema.SchemaObject { enum  = Just _ } -> mkPropertyType fieldSchema
+          Schema.SchemaObject{type' = Just _} -> mkPropertyType fieldSchema
+          Schema.SchemaObject{enum  = Just _} -> mkPropertyType fieldSchema
           schemaObject                           -> expandSum schemaObject
```
Comment (snusnu) [inline gen/src/Gen/Schema.hs]: @mbj The alignment needs to be fixed.

Comment (mbj) [reply gen/src/Gen/Schema.hs]: fixed, good catch.

**Pull request merged**